### DISCUSSION TITLE
demo挂了，500服务器内部错误

### DISCUSSION THREAD

**Discussion by rtsbtx** [halo-dev/halo#5576] [General]
### 系统信息

https://demo.halo.run/ 
挂了， 500服务器内部错误

### 使用的哪种方式运行？

Docker

### 发生了什么？

https://demo.halo.run/ 
挂了， 500服务器内部错误

**Comment by ruibaby** [comment]
目前已恢复，这通常是主题设置异常导致。

此 issue 不属于 Halo 本身问题，我将转移到 Discussion。

/triage unresolved
